rust_src/src/cmds.rs

```diff
@@ -534,6 +534,10 @@ pub extern "C" fn rust_syms_of_cmds() {
     def_lisp_sym!(Qoverwrite_mode_binary, "overwrite-mode-binary");
     def_lisp_sym!(Qexpand_abbrev, "expand-abbrev");
     def_lisp_sym!(Qpost_self_insert_hook, "post-self-insert-hook");
+
+    /// Hook run at the end of `self-insert-command'.
+    /// This is run after inserting the character.
```

Comment:
Do we extract docs from variables yet? In other words, does `C-h v post-self-insert-hook` work after this change?

I don't think it should hold up this PR if we don't do this yet though.

Reply:
Alas, we do not.